### PR TITLE
Document how to test exclusion patterns without performing an actual backup

### PR DESCRIPTION
**Pull request by miguno**

### CODE REVIEW

**Comment by miguno** [comment]
PR as suggested at https://github.com/borgbackup/borg/issues/4812#issuecomment-566528250.

File: docs/usage/help.rst.inc [hunk 1]
```diff
@@ -96,6 +96,9 @@ selector prefix is also supported for patterns loaded from a file. Due to
 whitespace removal paths with whitespace at the beginning or end can only be
 excluded using regular expressions.
 
+To test your exclusion patterns without performing an actual backup you can
+run ``borg create --list --dry-run ...``.
+
```
Comment (ThomasWaldmann) [inline docs/usage/help.rst.inc]: "IMPORTANT: this file is auto-generated from borg's built-in help, do not edit!" (1st line of this file).

You need to locate the same text in `src/borg/archiver.py` and change it there only, all other stuff will be auto-created from that at release time.

Comment (miguno) [reply docs/usage/help.rst.inc]: Ack, PR updated.

**Comment by miguno** [comment]
Travis CI build errors are unrelated to this PR.

```
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/trusty-updates/main/binary-amd64/Packages.gz  Hash Sum mismatch
```

```
The job exceeded the maximum time limit for jobs, and has been terminated.
```

**Comment by ThomasWaldmann** [comment]
Thanks! Do you want to make a PR for 1.1-maint also?

**Comment by miguno** [comment]
PR for `1.1-maint` up at https://github.com/borgbackup/borg/pull/4888.

Thanks for the quick merge, @ThomasWaldmann !

**Pull request merged**